chrome/content/zotero/containers/tagSelectorContainer.jsx

```diff
@@ -638,6 +638,49 @@ Zotero.TagSelector = class TagSelectorContainer extends React.PureComponent {
 		await Zotero.Tags.setColor(this.libraryID, io.name, io.color, io.position);
 	}
 
+	async openTagSplitterWindow() {
+		const oldTag = this.contextTag.name;
+		const dataIn = {
+			oldTag: this.contextTag.name,
+			isLongTag: false
+		};
+		const dataOut = { result: null };
+		
+		window.openDialog(
+			'chrome://zotero/content/longTagFixer.xhtml',
+			'',
+			'chrome,modal,centerscreen',
+			dataIn, dataOut
+		);
+
+		if (!dataOut.result) {
+			return;
+		}
+
+		const oldTagID = Zotero.Tags.getID(oldTag);
+
+		switch (dataOut.result.op) {
+			case 'split':
+				for (const library of Zotero.Libraries.getAll().filter(lib => lib.editable)) {
```

Comment:
This wouldn't be right. We'll only want to apply it to the current library. (If the same error is replicated elsewhere, you'll just need to apply it again — the tag selector is always library-specific.)